cli.js

```diff
@@ -8,6 +8,12 @@ const cli = meow(`
 	Usage
 	  $ package-json <name> [version=latest]
 
+	Options
+	  --full-metadata    --full      Output full package metadata
+	  --all-versions     --all       Output all versions
+	  --registry-url     --registry  Registry URL                  [Default: inferred]
```

Comment:
I would prefer not having aliases for these. These options are no common usage, and not something you type often.